CHANGELOG.md

```diff
@@ -2,6 +2,66 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.22.0 (In Progress)
+====================
+
+New Features
+------------
+
+This release include the **Media Warnings API (Beta)** to help surface media related warning events on the SDK whenever the media server is not able to detect media from a published audio or video track.
+
+### Example
+
+```js
+const room = connect('token', {
+ notifyWarnings: [ 'recordingMediaLost' ]
+ // Other connect options
+});
+
+room.on('trackWarning', (name, track, participant) => {
```

Comment:
I may have missed this during the IDL PR review: Since a LocalTrack is not associated with a Room per-se, I think we should raise the LocalTrackPublication as the second argument in the event.